hooks/post_gen_project.py

```diff
@@ -27,6 +27,13 @@ def remove_dir(path):
         remove_dir(".github")
         remove_dir(".dependabot")
 
+    if "{{ cookiecutter.open_source_project }}".lower() == "n":
+        remove_file("AUTHORS.md")
+        remove_file("CHANGELOG.md")
```

Comment:
Changelog is relevant also in case of non open source projects to keep track of what is actually part of each version. Ultimately that gets generated from the commit messages, but until that is in place this should be remained.

Reply:
Agree, i will put it back.